VERSION

```diff
@@ -0,0 +1 @@
+   1.4
```

Comment:
Let's use 1.4.1 as the version number to follow https://semver.org/

Reply:
done